csatar-plugins/knowledgerepository/models/WeeklyWorkPlan.php

```diff
@@ -0,0 +1,536 @@
+    }
+
+    public function getDistrict() {
+        return $this->patrol->team->district ?? null;
+    }
+
+    public function getTeam() {
+        return $this->patrol->team ? $this->team : null;
+    }
+
+    public function getTroop() {
+        return $this->patrol ? $this->patrol->troop : null;
+    }
+
+    public function getPatrol() {
+        return $this->patrol ?? null;
+    }
+
+    public static function getOrganizationTypeModelNameUserFriendly()
+    {
+        return Lang::get('csatar.knowledgerepository::lang.plugin.admin.weeklyWorkPlan.weeklyWorkPlan');
+    }
+
+    public function getActivitiesToAttach() {
+        if (empty($this->patrol)) {
+            return;
+        }
+
+        $specialAgeGroupId = $this->patrol->team->district->association->special_workplan_age_group_id ?? null;
+
+        if ($this->patrol && $specialAgeGroupId && $this->patrol->age_group->id == $specialAgeGroupId) {
+            $activitiesToAttach = [
+                'Nyitó szertartás' => [
+                    'sort_order' => 1,
+                    'default_duration' => 5,
+                ],
+                'Mozgós, erőkifejtős játék' => [
+                    'sort_order' => 2,
+                ],
+                'Mozgós játék' => [
+                    'sort_order' => 3,
+                ],
+                'Kevés mozgást igénylő játék, ügyességi, koncentrációs játék' => [
+                    'sort_order' => 4,
+                ],
+                'El nem mozdulós játék' => [
+                    'sort_order' => 5,
+                ],
+                'Új anyag' => [
+                    'sort_order' => 6,
+                ],
+                'Régi anyag' => [
+                    'sort_order' => 7,
+                ],
+                'Mozgós, szórakoztató játék' => [
+                    'sort_order' => 8,
+                ],
+                'Záró szertartás' => [
+                    'sort_order' => 9,
+                    'default_duration' => 5,
+                ]
+            ];
+        } else {
+            $activitiesToAttach = [
+                'Nyitó szertartás' => [
+                    'sort_order' => 1,
+                    'default_duration' => 5,
+                ],
+                'Játék' => [
+                    'sort_order' => 2,
+                ],
+                'Régi gyakorlat' => [
+                    'sort_order' => 3,
+                ],
+                'Új elmélet' => [
+                    'sort_order' => 4,
+                ],
+                'Dal' => [
+                    'sort_order' => 5,
+                    'default_duration' => 10,
+                ],
+                'Új gyakorlat' => [
+                    'sort_order' => 6,
+                ],
+                'Régi elmélet' => [
+                    'sort_order' => 7,
+                ],
+                'Új játék' => [
+                    'sort_order' => 8,
+                ],
+                'Záró szertartás' => [
```

Comment:
Ezeket a konstansokat nem lenne jobb vagy Seeder-be tenni, vagy valahova kimenteni?